templates/api_client.mustache

```diff
@@ -94,17 +94,17 @@ ApiClient  <- R6::R6Class(
       }
 
       if (method == "GET") {
-        httr::GET(url, query = queryParams, headers, httpTimeout, httr::user_agent(self$`userAgent`), ...)
+        httr::GET(url, query = queryParams, headers, httpTimeout, httr::user_agent(self$`userAgent`), httr::config(http_version=1), ...)
```

Comment:
https://github.com/jeroen/curl/issues/156 I was running into this:
```
HTTP/2 stream 0 was not closed cleanly: PROTOCOL_ERROR (err 1)
```
and so we're forcing requests to use HTTP/1.1 instead of allowing the HTTP/2 upgrade.